rgf/rgf.py

```diff
@@ -3,12 +3,21 @@
 from glob import glob
 import numpy as np
 from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin
-
+import platform
 ## Edit this ##################################################
 
+sys_name = platform.system()
+
 #Location of the RGF executable
-loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
-loc_temp = 'temp/'
+WINDOWS = 'Windows'
+LINUX = 'Linux'
+
+if sys_name == WINDOWS:
+    loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
+    loc_temp = 'temp/'
+elif sys_name == LINUX:
+    loc_exec = '/opt/rgf1.2/bin/rgf'
+    loc_temp = '/tmp'
```

Comment:
`loc_temp` is directory for placing rgf model cache.
Which would be better put always in the same directory (`tmp/`), or directory under the exec directory (`/tmp/`)?
I think the latter might result in unitended loading data.